src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Pledges.cshtml

```diff
@@ -46,6 +48,7 @@
                                 <td class="govuk-table__cell govuk-table__cell--numeric">£@pledge.Amount.ToString("N0")</td>
                                 <td class="govuk-table__cell govuk-table__cell--numeric">£@pledge.RemainingAmount.ToString("N0")</td>
                                 <td class="govuk-table__cell govuk-table__cell--numeric">@pledge.ApplicationCount</td>
+                                <td class="govuk-table__cell govuk-table__cell--numeric"><strong class="@pledge.Status.GetCssClassForPledgeStatus()">@pledge.Status.GetLabelForPledgeStatus()</strong></td>
```

Comment:
I have seen just govuk-table__cell being used for text fields at other places. May be we could do the same here?

Reply:
Sure, this is now updated.